screens/AlarmScreen.js

```diff
@@ -1,28 +1,55 @@
-import React, { Component } from "react";
-import {
-  StyleSheet,
-  Button
-} from "react-native";
+import React, { useEffect } from "react";
+import { StyleSheet, Alert } from "react-native";
+import { connect } from "react-redux";
 import { Actions } from "react-native-router-flux";
 import Timer from "../components/Timer/Timer"
-import { Container, Content, Header, View, Banner } from "../components/layout";
+import { Container, Content, Header, View } from "../components/layout";
+import { Button } from "../components/buttons"
 import { Text } from "../components/typography";
+import { sendHelpRequest } from "../services/help-request.service";
 
-const AlarmScreen = () => {
+const AlarmScreen = (props) => {
+
+  const { time, userId, token } = props
+  const { timeRemaining } = time
+
+  useEffect(() => {
+    // send help_request here
```

Comment:
What do you mean by this?

Reply:
Oh I think these were just remnants - I wanted to do an `if (timeRemaining === 0) sendHelpRequest` here